### PR TITLE
WT-11897 split the long running tests into two test tasks

### PR DESCRIPTION
**Pull request by jeremythorp**
Split the long running tests into two test tasks so they each complete within a 5-hour time limit.

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -2399,15 +2399,30 @@ tasks:
               ${python_binary|python3} ../test/suite/run.py -v 4 test_prepare_hs03.py --hook timestamp
             done
 
-  - name: csuite-long-running
+  # csuite-long-running-subset1 and csuite-long-running-subset2 test tasks each run half of the long-running tests
+  # so that each test task completes within the 5-hour time limit
+
+  - name: csuite-long-running-subset1
```
Comment (jiechenbo) [inline test/evergreen.yml:2405]: we usually use the name `bucket` when dividng the segments, would be clearer to stick to that style.

File: test/evergreen.yml [hunk 2]
```diff
@@ -5988,7 +6003,10 @@ buildvariants:
     - name: format-stress-pull-request-test
     - name: make-check-test
       distros: ubuntu2004-large
-    - name: csuite-long-running
+    - name: csuite-long-running-subset1
+      distros: ubuntu2004-large
```
Comment (jiechenbo) [inline test/evergreen.yml:6007]: is there a reason behind using a large ubuntu distro here?

File: test/evergreen.yml (hunk 1 above)
Comment (jeremythorp) [reply test/evergreen.yml:2405]: Good suggestion. I'll make that change.

File: test/evergreen.yml (hunk 2 above)
Comment (jeremythorp) [reply test/evergreen.yml:6007]: That was something I left unchanged from the previous version of the test.

Comment (jeremythorp) [reply test/evergreen.yml:6007]: It's worth seeing if those tests can run ok on a smaller distro, so I'll try that in a patch build.

Comment (jeremythorp) [reply test/evergreen.yml:6008]: I ran a patch build with a default (smaller) instance type, and one of the tests failed.
I've added a comment in each task explaining that a 'large' distro is required to run all the tests successfully.

**Review by jiechenbo — APPROVED** [review]
LGTM!

**Pull request merged**